support-frontend/assets/helpers/abTests/abtestDefinitions.ts

```diff
@@ -65,6 +65,26 @@ export const tests: Tests = {
 		seed: 2,
 		targetPage: pageUrlRegexes.contributions.allLandingPagesAndThankyouPages,
 	},
+	emotionalBenefits: {
+		variants: [
+			{
+				id: 'control',
+			},
+			{
+				id: 'variant',
+			},
+		],
+		audiences: {
+			ALL: {
+				offset: 0,
+				size: 0,
+			},
+		},
+		isActive: false,
+		referrerControlled: true,
+		seed: 5,
+		targetPage: pageUrlRegexes.contributions.allLandingPagesAndThankyouPages,
```

Comment:
@paul-daniel-dempsey before we enable this we'll need to target the US only in the `audiencies` property.

Reply:
Ah yes, thanks for the reminder, will add a `UnitedStates` this when goes live or if need to do an amendment PR